ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -26089,7 +26089,7 @@ periodics:
     repo: release
   labels:
     ci-operator.openshift.io/cloud: packet
-    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/cloud-cluster-profile: packet
```

Comment:
@derekhiggins is this ok (not very clear from this diff)?

Reply:
Yes, I switched one job(nightly-4.6, e2e-metal-ipi ) back to packet in 
 job in ci-operator/config/openshift/release/openshift-release-master__nightly-4.6.yaml

Reply:
Thanks